pivy-importer/src/main/groovy/com/linkedin/python/importer/ImporterCLI.java

```diff
@@ -57,26 +59,45 @@ public static void main(String[] args) throws Exception {
             throw new RuntimeException("Unable to continue, no repository location given on the command line (use the --repo switch)");
         }
         final File repoPath = new File(line.getOptionValue("repo"));
-        final DependencySubstitution replacements = new DependencySubstitution(buildSubstitutionMap(line), buildForceMap(line));
 
         repoPath.mkdirs();
 
         if (!repoPath.exists() || !repoPath.isDirectory()) {
             throw new RuntimeException("Unable to continue, " + repoPath.getAbsolutePath() + " does not exist, or is not a directory");
         }
 
+        importPackages(line, repoPath);
+        logger.info("Execution Finished!");
+    }
 
+    private static void importPackages(CommandLine line, File repoPath) {
+        final DependencySubstitution replacements = new DependencySubstitution(buildSubstitutionMap(line), buildForceMap(line));
         Set<String> processedDependencies = new HashSet<>();
         for (String dependency : line.getArgList()) {
-            DependencyDownloader dependencyDownloader = new DependencyDownloader(
-                    dependency, repoPath, replacements, line.hasOption("latest"), line.hasOption("pre"),
-                    line.hasOption("lenient"));
-            dependencyDownloader.getProcessedDependencies().addAll(processedDependencies);
-            dependencyDownloader.download();
-            processedDependencies.addAll(dependencyDownloader.getProcessedDependencies());
-        }
+            DependencyDownloader artifactDownloader;
+
+            if (dependency.split(":").length == 2) {
+                artifactDownloader = new SdistDownloader(dependency, repoPath, replacements, line.hasOption("latest"),
+                    line.hasOption("pre"), line.hasOption("lenient"));
+            } else if (dependency.split(":").length == 3) {
```

Comment:
Instead of looking at the number of arguments, should we just use a -wheel argument?

Reply:
I thought about that, if we use "--wheel", there are two problems:
1. User needs to specify "--wheel" which adds more input parameters, but we can actually just find out if they want to get wheels or sdist by the value of classifiers, in this way users do not need to specify an additional parameter.
2. If we use "--wheel", then each command user can only get sdist or wheels, but without "--wheel", as you can see in the description, we can get the sdist and wheels in one command.

Reply:
You convinced me. Let's keep it the way it is!

Reply:
So users can never get *just* the wheel?

Reply:
We give them the flexibility to import any case they want, which means they can
1. Only import sdist
2. Only import wheels by specify classifier
3. import both. (pivy-importer can tell if it is sdist or wheels by classifier)

Reply:
Ok I misread this earlier. Looks good.